components/tyk-dashboard/values.yaml

```diff
@@ -373,6 +373,8 @@ dashboard:
     # tls corresponds to the tls configuration if Ingress rules use TLS
     tls: []
 
+  serviceAccountName: ""
```

Comment:
Please add comment about what is this field, what happens if users configure this field or what happens if they do not provide this field.